end_users/README.md

```diff
@@ -1,3 +1,24 @@
 # End Users
 
-Documentation coming soon...
+## Table of Contents
+
+* [Merge Options](#merge-options)
+  * [Setting the Correct Merge Option](#setting-the-correct-merge-option)
+
+## Merge Options
+
+Repositories that use BonnyCI should not use any merge strategy which changes the SHA of the commit. This is because [Zuul](https://github.com/openstack-infra/zuul), the scheduler used in BonnyCI, will be thrown off by the change in SHA when it merges commits that successfully pass the gate queue.
```

Comment:
I feel like it's a good idea to at least clarify what zuul is and how it relates to Bonny CI. This is end-user documentation, which means that the reader may not have read the developers section and may not know how zuul fits into the workflow.